e2e/scaffolding/helpers.ts

```diff
@@ -461,19 +487,26 @@ export async function setEpochLength(keys: KeyringPair, epochLength: number): Pr
   }
 }
 
+export async function getNextRewardEraBlock(): Promise<number> {
+  const eraInfo = await ExtrinsicHelper.apiPromise.query.capacity.currentEraInfo();
+  const actualEraLength: number = 50;
+  return actualEraLength + eraInfo.startedAt.toNumber() + 1;
+}
+
 export async function getOrCreateGraphChangeSchema(source: KeyringPair): Promise<u16> {
   const existingSchemaId = getGraphChangeSchema();
   if (existingSchemaId) {
     return new u16(ExtrinsicHelper.api.registry, existingSchemaId);
   } else {
-    const { target: createSchemaEvent, eventMap } = await ExtrinsicHelper.createSchemaV3(
+    const op = ExtrinsicHelper.createSchemaV3(
```

Comment:
Wondering if this change is real or a rebase mistake.

Reply:
It is fine either way, so OK by me at least.